### DISCUSSION TITLE
Extract Data and transfer it into csv

### DISCUSSION THREAD

**Discussion by bilel910** [mindee/doctr#1530] [Q&A]
Hello guys i worked with DocTr to extract data from images it works fine but the result i can just put it into a json file due to the structure i did some work where i deleted some fileds like : fields_to_remove = ['confidence', 'page_idx', 'dimensions', 'orientation', 'language', 'artefacts'] now my problem how to convert the result into a csv where i extract table from images that have specific format 
![image](https://github.com/mindee/doctr/assets/83314544/3699ce1e-7909-4726-9a12-24ced0c5f5db)

**Comment by felixdittrich92** [comment]
Hi @bilel910 :wave:,

You could pass `resolve_blocks=False` to the `ocr_predictor` instance so you can work with the output on line level.
Or you combine https://huggingface.co/docs/transformers/model_doc/table-transformer for structured table extraction with docTR as OCR engine. :)

**Reply by felixdittrich92** [reply]
There seems to be already an package which combines both: https://github.com/deepdoctection/deepdoctection

**Reply by bilel910** [reply]
thank you i'll try it
